### PR TITLE
Missing fork or bear instruction

### PR DESCRIPTION
**Pull request by dgearhart**
# Issue

Missing fork or bear instruction which could cause reroutes for the user.

---
### Example 1

image
``` DIFF
< BEFORE
< 5: Continue on A6182/White Rose Way.
<    5: TURN_LANES: [ left | left | through ACTIVE | through ACTIVE ] ACTIVE_TURN_LANES
> AFTER
> 5: Keep left to stay on A6182/White Rose Way toward Doncaster/Railport/Stadium/Racecourse.
>    5: TURN_LANES: [ *left* ACTIVE | *left* ACTIVE | through | through ] ACTIVE_TURN_LANES
```
image

---
### Example 2

image
``` DIFF
< BEFORE
< 1: Drive northwest on 9th Street.
> AFTER
> 1: Drive northwest on 9th Street.
> 2: Bear right onto Larkin Street.
```
image

---
### Example 3
image
``` DIFF
< BEFORE
< 1: Drive northwest on 9th Street.
> AFTER
> 1: Drive northwest on 9th Street.
> 2: Bear left onto Hayes Street.
```
image

---
### Example 4
image
``` DIFF
< BEFORE
< 1: Drive south on A38/Devon Expressway.
> AFTER
> 1: Drive south on A38/Devon Expressway.
> 2: Keep left to take A380 toward Torquay.
>    2: TURN_LANES: [ *slight_left* ACTIVE | *slight_left* ACTIVE | slight_right | slight_right ] ACTIVE_TURN_LANES
```
image

---
### Example 5
image
``` DIFF
< BEFORE
< 1: Drive south on A38/Devon Expressway. Continue on A38.
> AFTER
> 1: Drive south on A38/Devon Expressway.
> 2: Keep right to take A38 toward Plymouth.
>    2: TURN_LANES: [ slight_left | slight_left | *slight_right* ACTIVE | *slight_right* ACTIVE ] ACTIVE_TURN_LANES
```
image

---
### Example 6
image
``` DIFF
< BEFORE
< 1: Drive east on 퇴계로/Toegye-ro.
> AFTER
> 1: Drive east on 퇴계로/Toegye-ro.
> 2: Bear right onto 소파로/Sopa-ro.
> 3: Keep left to take 삼일대로/Samil-daero.
```

---
## Tasklist

 - [x] Add tests
 - [x] Add #fixes with the issue number that this PR addresses
 - [x] Generally use squash merge to rebase and clean comments before merging
 - [x] Update the [changelog](CHANGELOG.md)

### CODE REVIEW

File: src/odin/maneuversbuilder.cc [hunk 1]
```diff
@@ -1471,7 +1474,7 @@ void ManeuversBuilder::SetSimpleDirectionalManeuverType(Maneuver& maneuver,
         }
         ////////////////////////////////////////////////////////////////////
         // If internal intersection at beginning of maneuver
-        else if (curr_edge && curr_edge->internal_intersection()) {
```
Comment (gknisely) [inline src/odin/maneuversbuilder.cc:1474]: no check for valid curr_edge needed?

Comment (gknisely) [reply src/odin/maneuversbuilder.cc:1474]: nm i see above

File: src/odin/narrativebuilder.cc [hunk 2]
```diff
@@ -4217,7 +4265,7 @@ std::string NarrativeBuilder_csCZ::GetPluralCategory(size_t count) {
 }
 
 ///////////////////////////////////////////////////////////////////////////////
-std::string NarrativeBuilder_hiIN::GetPluralCategory(size_t count) {
+std::string NarrativeBuilder_hiIN::GetPluralCategory(size_t /*count*/) {
```
Comment (mandeepsandhu) [inline src/odin/narrativebuilder.cc:4268]: Is this argument for future use? If not, can we remove it?

File: test/gurka/test_instructions_keep_highway_signs.cc [hunk 3]
```diff
@@ -94,16 +93,15 @@ TEST_F(InstructionsKeepToward, KeepRightToward) {
   // Verify the keep right toward instructions
   gurka::assert::raw::
       expect_instructions_at_maneuver_index(result, maneuver_index,
-                                            "Keep right toward B2/Baltimore/Washington.",
-                                            "Keep right toward B2.",
-                                            "Keep right toward B2, Baltimore.",
+                                            "Keep right to take B2 toward Baltimore/Washington.",
```
Comment (mandeepsandhu) [inline test/gurka/test_instructions_keep_highway_signs.cc:96]: This reads a lot better! 👏 ty!

File: test_requests/fork_bear_routes.txt [hunk 4]
```diff
@@ -0,0 +1,50 @@
+-j '{"locations":[{"lat":53.487484,"lon":-1.126424,"street":"M18"},{"lat":53.506649,"lon":-1.119099,"street":"A6182"}],"costing":"auto","units":"kilometers"}'
```
Comment (mandeepsandhu) [inline test_requests/fork_bear_routes.txt:1]: Thanks for adding these new tests to the corpus!

**Review by mandeepsandhu — APPROVED** [review]

File: src/odin/narrativebuilder.cc (hunk 2 above)
Comment (dgearhart) [reply src/odin/narrativebuilder.cc:4268]: yes, it might be used depending on the rules for each language

**Pull request merged**